lib/core/src/lib/directives/node-delete.directive.spec.ts

```diff
@@ -271,16 +271,16 @@ describe('NodeDeleteDirective', () => {
             });
         });
 
-        it('should emit event when delete is done', (done) => {
+        it('should emit event when delete is done', async () => {
```

Comment:
you make it async but not using await in the code, I suggest converting observable to promise and awaiting it

Reply:
await added